#! /bin/sh /usr/share/dpatch/dpatch-run
#
# DP: From: Niko Tyni <ntyni@debian.org>
# DP: Date: Fri, 11 Dec 2009 13:12:46 +0200
# DP: Subject: [PATCH] Accommodate to HTML::Parser 3.64 change in the file open interface.

@DPATCH@

2009-10-25  Release 3.64
[...]
      Make parse_file() method use 3-arg open [RT#49434]

Using \*STDIN should work with both the old and the new versions.
---
 util/fix-pod2html.pl |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/util/fix-pod2html.pl b/util/fix-pod2html.pl
index fa51400..ff0176e 100755
--- a/util/fix-pod2html.pl
+++ b/util/fix-pod2html.pl
@@ -22,7 +22,7 @@ my $p = HTML::Parser->new(api_version => 3);
 $p->handler(start => \&startsub, 'tagname, text');
 $p->handler(end => \&endsub, 'tagname, text');
 $p->handler(default => sub { print shift() }, 'text');
-$p->parse_file(shift||"-") or die("parse: $!");
+$p->parse_file(shift||\*STDIN) or die("parse: $!");
 
 my @stack;
 my $a=0;
-- 
1.6.5.4

